.husky/pre-commit

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Skip pre-commit hook on CI
+[ -n "$CI" ] && exit 0
+
+npx --no-install lint-staged
```

Comment:
Since `lint-staged` is already installed, the `--no-install` option isn't necessary

```suggestion
npx lint-staged
```